NUCLEO_F439ZI/Src/main.c

```diff
@@ -169,7 +169,12 @@ int main(void)
     BSP_LED_Off(LED2);
     HAL_Delay(500);
   }
-  return 0;
+
```

Comment:
I intentionally didn't put this in the STM32 main as:
- the return value is not consumed by anything,
- there is an endless loop right before this, so this code is never reached,
- we try to keep libtropic related code inside the "libtropic related code BEGIN/END".

So let's keep dummy "return 0".

Reply:
Sure, I think this solution would be clearer.